libs/kinematics/src/CVehicleSimul_Holo.cpp

```diff
@@ -56,22 +35,22 @@ void CVehicleSimul_Holo::internal_simulStep(const double dt)
 		// "Blending" for vx,vy
 		if (t<=m_vel_ramp_cmd.ramp_time)
 		{
-			m_vel.vx = vxi + t*(vxf-vxi) /T;
-			m_vel.vy = vyi + t*(vyf-vyi) /T;
+			m_odometric_vel.vx = vxi + t*(vxf-vxi) /T;
+			m_odometric_vel.vy = vyi + t*(vyf-vyi) /T;
 		}
 		else
 		{
-			m_vel.vx = m_vel_ramp_cmd.target_vel_x;
-			m_vel.vy = m_vel_ramp_cmd.target_vel_y;
+			m_odometric_vel.vx = m_vel_ramp_cmd.target_vel_x;
+			m_odometric_vel.vy = m_vel_ramp_cmd.target_vel_y;
 		}
 #if 0
 		// Proportional controller in angle:
 		//const double KP = 1.0;
-		//m_vel.phi = KP* mrpt::math::wrapToPi(m_vel_ramp_cmd.dir - m_pose.phi) * m_vel_ramp_cmd.rot_speed;
+		//m_vel.phi = KP* mrpt::math::wrapToPi(m_vel_ramp_cmd.dir - m_GT_pose.phi) * m_vel_ramp_cmd.rot_speed;
 #else
 		// Constant rotational velocity:
-		const double Aang = mrpt::math::wrapToPi(m_vel_ramp_cmd.dir - m_pose.phi);
-		m_vel.omega = ( std::abs(Aang)<mrpt::utils::DEG2RAD(1.0) ? 0.0 : mrpt::utils::sign(Aang) )  * std::abs(m_vel_ramp_cmd.rot_speed);
+		const double Aang = mrpt::math::wrapToPi(m_vel_ramp_cmd.dir - m_odometry.phi);
+		m_odometric_vel.omega = ( std::abs(Aang)<mrpt::utils::DEG2RAD(1.0) ? 0.0 : mrpt::utils::sign(Aang) )  * std::abs(m_vel_ramp_cmd.rot_speed);
 #endif
```

Comment:
Please, add a MRPT_TODO() here to remind to me that I must fix this to emulate the rotvel ramp time as well...